source/meetings/2019/march/index.html.md

```diff
@@ -0,0 +1,95 @@
+---
+updated_by:
+  email: ella.schofield@lrug.org
+  name: Ella Schofield
+created_by:
+  email: ella.schofield@lrug.org
+  name: Ella Schofield
+category: meeting
+title: March 2019 Meeting
+updated_at: 2019-02-22 16:03:34 +0000
+published_at: 2019-02-22 16:03:34 +0000
+created_at: 2019-02-22 16:03:34 +0000
+status: Published
+---
+
+The March 2019 meeting of LRUG will be on *Monday the 11th of March*,
+from _6:00pm_ to _8:00pm_ (meeting start at _6:30pm_).  The venue, [Code
```

Comment:
This may have slipped through the net but in https://lrug.zendesk.com/inbox/conversations/341 Skills Matter let us know that there'll be a 30 minute delay.  E.g. Doors at 6:30, start at 7.

Reply:
That said, they've put it up on their site as starting at 6:30.  We should reply to that conversation and check the start time, and get them to update accordingly if it's wrong.